### PR TITLE
Debug/slow upgrade tests

### PR DESCRIPTION
**Pull request by dcapwell**

### CODE REVIEW

File: dtest.py [hunk 1]
```diff
@@ -249,6 +249,12 @@ def set_dtest_setup_on_function(self, fixture_dtest_setup):
         self.dtest_config = fixture_dtest_setup.dtest_config
         return None
 
+    @pytest.fixture(autouse=True)
+    def cleanup_connections(self):
+        yield None
```
Comment (aratno) [inline dtest.py:254]: Can you add a comment about why this yield is necessary?

**Review by aratno — APPROVED** [review]

File: dtest.py (hunk 1 above)
Comment (dcapwell) [reply dtest.py:254]: this is documented when you look up `fixture`, its a core way that this works... w/e is before `yield` is setup, w/e is after `yield` is cleanup.

Comment (aratno) [reply dtest.py:254]: Ah right on - no extra doc necessary then

Comment (dcapwell) [reply dtest.py:254]: https://docs.pytest.org/en/6.2.x/fixture.html#yield-fixtures-recommended

**Pull request closed without merging**